docs/REST-API/99-Rate-Limiting-23.md

```diff
@@ -0,0 +1,74 @@
+
+**Account API Keys**
+
+Each Account API Key is allowed 960 requests per minute.
+
+**User API Keys**
+
+User API Keys act as a PagerDuty User and are limited as such. Each user is allowed 960 requests per minute across all of the user's API keys.
+
+**PagerDuty App - App OAuth Tokens**
+
+When a PagerDuty App [uses an app token](/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty App. Each PagerDuty App is allowed 960 requests per minute against each PagerDuty Account it is authorized to access.
+
+**PagerDuty App - User OAuth Tokens**
+
+When a PagerDuty App [uses a user token](/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty User. Each PagerDuty App is allowed 960 requests per minute per User it is authorized to act as.
+
+### Operation specific limits
+In a few exceptional circumstances, individual REST API operations may apply additional limits. If an endpoint applies additional limits; it will be noted in the [REST API](/api-reference/) documentation for that endpoint. The ratelimit response headers will provide information to the client about the current status of the limit and when it resets.
+
+## Working with limits
+Okay so REST API rate limits are a thing. How can your application gracefully interact with these limits? This section describes how our API informs client applications about limiting and covers some best practices.
+
+### Rate limit headers
+Requests that have been evaluated for rate limiting will include the following HTTP response headers:
+
+|Header Name|Description|
+|-|-|
+|`ratelimit-limit`|The allowed number of requests for the current time window.|
+|`ratelimit-remaining`|The remaining number requests in the current time window before limiting occurs.|
+|`ratelimit-reset`|The number of seconds until the limit is reset.|
+
+### Reaching the limit
+In the event that your application reaches a limit: the HTTP response code will be `429`, the rate limit headers will be present, and the response body will indicate that limiting has occurred:
+
+```bash
+HTTP/2 429 Too Many Requests
+
+date: Tue, 03 Oct 2023 16:46:39 GMT
+ratelimit-limit: 960
+ratelimit-remaining: 0
+ratelimit-reset: 20
+
+{
+  "error": {
+    "message": "Rate Limit Exceeded",
+    "code": 2020
+  }
+}
+```
+
+### Best practices
+
+#### Make use of the rate limit headers
+Update your application to make use of the `ratelimit-remaining` and `ratelimit-reset` response headers. If your application is being limited; make use of the `ratelimit-reset` to have your application wait the appropriate
+number of seconds before retrying.
+
+#### One "bot user" per application deployment
+If you use PagerDuty Users / User API Keys as "bot users" for applications; create a separate bot user for each deployment of the application (e.g. "Acme Production Bot User", "Acme Test Bot User", etc.). This will keep the limits for each deployment of the application separate.
```

Comment:
> as "bot users" for applications; create a separate

Probably a comma here instead.

Reply:
I think it is supposed to be a semicolon.